modifiers/ReplaceBlockPairs.groovy

```diff
@@ -2,16 +2,15 @@ import net.querz.mcaselector.io.mca.ChunkData
 import net.querz.nbt.CompoundTag
 
 /**
- * Defines pairs of blocks to replace.
+ * Specify blocks as comma-separated pairs in the format 'source:original_block': 'source:new_block'
  * Replaces the original block in each block pair with its respective destination block.
- * Note: Block states are not updated, only the block name is changed.
  *
+ * Note: Block states are not updated, only the block name is changed.
  * @type Change NBT (Ctrl + N)
  * @difficulty INTERMEDIATE
  */
 static void apply(ChunkData data) {
     def blockpairs = ['byg:red_rock': 'byg:travertine', 'byg:aspen_log': 'byg:pendorite_block']
-//block replacements are specified here as comma-separated pairs in the format 'source:original_block': 'source:new_block'
```

Comment:
actually, I just meant to put this line above the `def blockpairs...` instead of below. It's good to have a comment directly above the line where a user would change the values, that makes it more easy to understand.

Reply:
Oh, okay. I think I've got all your edits.